__tests__/UrqlResponse_test.re

```diff
@@ -85,64 +85,4 @@ describe("UrqlResponse", () => {
       Expect.(expect(result.response) |> toEqual(UrqlTypes.NotFound));
     });
   });
-
-  describe("urqlClientResponseToReason", () => {
```

Comment:
This function got moved into `UrqlClient` since it's only in use by that module.